<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomSearch</class>
 <widget class="QMainWindow" name="CustomSearch">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>768</width>
    <height>442</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Search</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>140</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  display: inline-block;
  border-radius: 4px;
  background-color: #f4511e;
  border: none;
  color: #FFFFFF;
  text-align: center;
  font-size: 16px;
  padding: 5px;
  width: 100px;
  transition: all 0.5s;
  cursor: pointer;
  margin: 5px;
}</string>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>190</y>
      <width>611</width>
      <height>201</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Date</string>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Flight Name</string>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Departure TIme</string>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>BC Seats</string>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>XC Seats </string>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>EC Seats </string>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
    </column>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>130</y>
      <width>181</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Andalus</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Also show next 7 days scedule</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Andalus</family>
     </font>
    </property>
    <property name="text">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Andalus'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:18pt; font-weight:600; color:#aaffff;&quot;&gt;Search&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>70</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Andalus</family>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; color:#000000;&quot;&gt;Origin:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="pic2_label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>761</width>
      <height>401</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../Downloads/airline/back.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>100</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>100</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Select</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>India</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>United States</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>UK</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Dubai</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Turkey</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>100</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Select</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>India</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>United States</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>UK</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Dubai</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Turkey</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>70</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Andalus</family>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; color:#000000;&quot;&gt;Destination:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>70</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Andalus</family>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; color:#000000;&quot;&gt;Journey Date:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
  display: inline-block;
  border-radius: 4px;
  background-color: green;
  border: none;
  color: #FFFFFF;
  text-align: center;
  font-size: 16px;
  padding: 5px;
  width: 100px;
  transition: all 0.5s;
  cursor: pointer;
  margin: 5px;
}</string>
    </property>
    <property name="text">
     <string>‚Üê Back</string>
    </property>
   </widget>
   <zorder>pic2_label</zorder>
   <zorder>pushButton</zorder>
   <zorder>tableWidget</zorder>
   <zorder>checkBox</zorder>
   <zorder>label</zorder>
   <zorder>label_8</zorder>
   <zorder>dateEdit</zorder>
   <zorder>comboBox</zorder>
   <zorder>comboBox_2</zorder>
   <zorder>label_7</zorder>
   <zorder>label_9</zorder>
   <zorder>pushButton_2</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>768</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
